test/unittests/statetest_loader_tx_test.cpp

```diff
@@ -0,0 +1,80 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+#include <test/statetest/statetest.hpp>
+
+using namespace evmone;
+
+TEST(statetest_loader, tx_create_legacy)
```

Comment:
Should probably add a test where both `gasPrice` and `maxFeePerGas` is present.

Reply:
What should happen then?

Reply:
Just fail, pushed my version.